### PR TITLE
medium logo added and few minor changes

### PR DESCRIPTION
**Pull request by harshs14**
Added medium icon, updated font awesome CDN and updated the icons respectively.
Also updated the link for si logo for google search and did few minor changes.

@bvishal8510 @deepanshu-jain1999 @SukhnandanMalhotra please review

### CODE REVIEW

**Review by SukhnandanMalhotra — APPROVED** [review]

File: Core/validators.py [hunk 1]
```diff
@@ -6,4 +6,3 @@ def validate_file_extension(value):
     ext = os.path.splitext(value)[1]
     if ext.lower() != '.pdf':
         raise ValidationError(u'File extension should be pdf!')
-
```
Comment (bvishal8510) [inline Core/validators.py:9]: according to PEP8, there should be a empty line at the end of file

File: IceCream/settings/base.py [hunk 2]
```diff
@@ -135,4 +135,4 @@
 EMAIL_HOST_PASSWORD = os.environ['EMAIL_PASSWORD']
 RECEIVER_EMAIL = os.environ['RECEIVER_EMAIL']
 EMAIL_USE_TLS = True
-DEFAULT_FROM_EMAIL = os.environ['SENDER_EMAIL']
\ No newline at end of file
+DEFAULT_FROM_EMAIL = os.environ['SENDER_EMAIL']
```
Comment (bvishal8510) [inline IceCream/settings/base.py:138]: here also empty line

File: Core/urls.py [hunk 3]
```diff
@@ -1,12 +1,12 @@
-from .views import *
+from .views import IndexView, RegistrationView, BlogView, BlogDetailView, SaveContactView
 from django.conf.urls import url
 
 urlpatterns = [
     url(r'^$', IndexView.as_view(), name='home'),
     url(r'^savecontact$', SaveContactView.as_view(), name='save_contact'),
     url(r'^register', RegistrationView.as_view(), name='registration'),
     url(r'^blogs', BlogView.as_view(), name='blog'),
-    url(r'^blogs/(?P<pk>\d+)/detail', BlogDetailView.as_view(), name='blog_detail'),
```
Comment (bvishal8510) [inline Core/urls.py]: it is a good convention to have comma(,) at the end of last element.

**Comment by harshs14** [comment]
@bvishal8510 sir all changes are done, pls review

**Pull request merged**